templates/terraform/resource.html.markdown.erb

```diff
@@ -46,10 +45,10 @@ To get more information about <%= object.name -%>, see:
 <%   end # object...guides.empty? -%>
 <% end # object...api.nil? -%>
 
-<% if documentation&.key?("examples") -%>
+<% unless object.documentation.examples.nil? -%>
```

Comment:
You can use &. if your values can be missing like here: `object&.examples&.nil?`

Reply:
Not needed because I removed the extra nesting